tools/starman.psgi

```diff
@@ -6,7 +6,8 @@ BEGIN {
 
 package LedgerSMB::FCGI;
 
-use lib 'lib';
+use FindBin;
+use lib $FindBin::Bin . "/../lib";
```

Comment:
As a note, I am looking at
1.  Removing the implicit load from Sysconfig and requiring a path passed in to an API when we want to load.
2.  Removal of other relative directories in PSGI.pm

That should improve both testability and safety.  I have the second working but the first is giving me some trouble